pkgs/servers/bazel-remote/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage {
```

Comment:
is this duplicating #55187? Can we please merge back into #54996 but have 3 separate commits? It's difficult to review or test the PRs as it.